content/docs/lists-and-keys.md

```diff
@@ -281,6 +280,7 @@ function NumberList(props) {
 ```
 
 JSX allows [embedding any expression](/docs/introducing-jsx.html#embedding-expressions-in-jsx) in curly braces so we could inline the `map()` result:
+JSX कर्ली ब्रेसिज़ में किसी भी एक्सप्रेशन को एम्बेडिंग करने की अनुमति देता है ताकि हम `map()` के परिणाम को इनलाइन कर सकें:
```

Comment:
`एक्सप्रेशन को एम्बेडिंग` => `[एक्सप्रेशन की एम्बेडिंग](/docs/introducing-jsx.html#embedding-expressions-in-jsx)`